openshift/template.yaml

```diff
@@ -155,16 +160,17 @@ objects:
           labels:
             glitchtip: web
         spec:
+          serviceAccountName: glitchtip-web-sa
           affinity:
             podAntiAffinity:
               requiredDuringSchedulingIgnoredDuringExecution:
-                - labelSelector:
-                    matchExpressions:
-                      - key: glithtip
```

Comment:
@vinaybommana a small fix is needed the key glitch-tip is spelled wrong you have `glithtip` instead of `glitchtip`